runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldTableRow.java

```diff
@@ -232,6 +232,11 @@ int getPing()
 		return ping;
 	}
 
+	boolean worldHasActivity()
+	{
+		return !world.getActivity().equals("-");

```

Comment:
Another option is to remove this function and have the following in the other file instead.

```java
String a1 = r1.getWorld().getActivity();
String a2 = r2.getWorld().getActivity();
return a1.equals("-") == a2.equals("-")
	? a1.compareTo(a2) * (ascendingOrder ? 1 : -1)
	: a1.equals("-") ? 1 : -1;
```